android/src/main/java/guichaguri/trackplayer/player/players/ExoPlayback.java

```diff
@@ -105,6 +105,10 @@ public void reset() {
     public void play() {
         player.setPlayWhenReady(true);
     }
+    @Override
+    public void playWithEarPiece() {
+        player.setPlayWhenReady(true);
```

Comment:
No support for ExoPlayer?
The "Android Player" is going to be removed in a future version, as ExoPlayer is much more reliable

Reply:
Will take a look on this. Thanks